Configure

```diff
@@ -462,7 +462,6 @@ our %disabled = ( # "what"         => "comment"
                   "ssl-trace"           => "default",
                   "ssl3"                => "default",
                   "ssl3-method"         => "default",
-                  "trace"               => "default",
```

Comment:
The trace api is disabled by default for good reasons, see 6bc62a620e7, #8474 and #8472. Please **don't** change that.

I would accept if 'debug' woud imply 'trace', but I don't see a precedent in the Configure file, i.e., there is no `@enable_cascades` analogous to `@disable_cascades`. 

https://github.com/openssl/openssl/blob/2c8a40777000cd7f3ddbe2c17c939d48ae1f805f/Configure#L474-L478

Reply:
> I would accept if 'debug' would imply 'trace'

I'm not even sure whether such an automatism needs to be added at all. Since most people either rely on command line history or have personal configure scripts, the trace option can easily be added to the command line.

Reply:
DO NOT ENABLE TRACE BY DEFAULT.

Reply:
Thank you for this feedback.
David is on vacation the next two weeks, but he will change this as soon as possible.

Reply:
No hurry. I‘ll be on vacation, too.